### PR TITLE
enable the dotnet tests and make the test run not required

### PR DESCRIPTION
**Pull request by elliotCamblor**

### CODE REVIEW

File: .github/workflows/config.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ jobs:
   run-tests:
     runs-on: ubuntu-latest
     name: Run Tests
+    continue-on-error: true
```
Comment (ajwootto) [inline .github/workflows/config.yml:19]: whats this for

Comment (elliotCamblor) [reply .github/workflows/config.yml:19]: thats supposed to make it optional I thought

Comment (elliotCamblor) [reply .github/workflows/config.yml:19]: nvm I misunderstood the docs

Comment (ajwootto) [reply .github/workflows/config.yml]: making it optional is just a repository setting

Comment (elliotCamblor) [reply .github/workflows/config.yml]: ah, I dont have permission which is why I cant see the setting

**Review by ajwootto — APPROVED** [review]

**Pull request merged**